methods.go

```diff
@@ -83,3 +70,28 @@ func EnableColor(flag bool) {
 func ListLoggersNames() []string {
 	return []string{prettylogger.LoggerName, zaplogger.LoggerName, nonelogger.LoggerName}
 }
+
+// InitOtel configures OpenTelemetry to export data to OTEL_COLLECTOR_SVC using uptrace collector.
+// You have to set the env variable OTEL_COLLECTOR_SVC to enable otel.
+// It is required to call ShutdownOtel on the context at the end of the main.
+func InitOtel(log helpers.ILogger, service, version string) context.Context {
+	ctx := context.Background()
+	if svc, ok := os.LookupEnv("OTEL_COLLECTOR_SVC"); ok {
+		var attrs []attribute.KeyValue
+		if id, ok := os.LookupEnv("ACCOUNT_ID"); ok {
+			attrs = append(attrs, attribute.String("account.id", id))
+		}
+		uptrace.ConfigureOpentelemetry(
+			uptrace.WithDSN(fmt.Sprintf("http://t@%s/1", svc)),
+			uptrace.WithServiceName(service),
+			uptrace.WithServiceVersion(version),
+			uptrace.WithResourceAttributes(attrs...),
+		)
+		log.Info("Otel metrics enabled, sending to collector " + svc)
```

Comment:
```suggestion
		log.Info("Otel metrics enabled", helpers.String("sending to collector", svc)
```

[Required]: Do not use a different logger :)
If we cannot print here the logs, we

Reply:
what do you mean? this is the logger given as an argument, so it matches the application one